litmus-portal/frontend/src/utils/yamlUtils.ts

```diff
@@ -29,7 +29,7 @@ const parsed = (yaml: string) => {
     const parsedYaml = YAML.parse(file as string);
     try {
       const embeddedYaml =
-        parsedYaml.spec.workflowSpec.templates[1].inputs.artifacts[0].raw.data;
```

Comment:
Why thus change required?

Reply:
The parsing is only possible for workflows not cron workflows so we have to change it to test this.

Reply:
Currently, we don't have cron workflow Right? You can raise it in a separate PR.

Reply:
Yeah this is required as the parser was for cron workflow, i have changed it to normal workflow, without this change the frontend will break. @rajdas98